code/game/objects/structures/mineral_doors.dm

```diff
@@ -418,6 +426,17 @@
 					obj_integrity = max_integrity
 				user.visible_message(span_notice("[user] repaired [src]."), \
 				span_notice("I repaired [src]."))		
+/obj/structure/mineral_door/attack_right(mob/user)
+	user.changeNext_move(CLICK_CD_FAST)
```

Comment:
By default there appears to be a 0.1s click cooldown- this changes it to 0.9(0.8 for fast+0.1 base). What's the purpose of this change?